manifests/input.pp

```diff
@@ -105,8 +105,13 @@
     }
 
     'Windows' : {
-      $cmd_install_dir = regsubst($filebeat::install_dir, '/', '\\', 'G')
-      $filebeat_path = join([$cmd_install_dir, 'Filebeat', 'filebeat.exe'], '\\')
+      if $filebeat::real_package_provider == 'exec' {
+        $cmd_install_dir = regsubst($filebeat::install_dir, '/', '\\', 'G')
+        $filebeat_path = join([$cmd_install_dir, 'Filebeat', 'filebeat.exe'], '\\')
+      }
+      else {
+        $filebeat_path = 'C:/ProgramData/chocolatey/lib/filebeat/tools/filebeat.exe'
```

Comment:
"$env:ChocolateyInstall/bin/filebeat.exe" (needs ruby code for environment) - also I'm pointing to the shim in the bin folder, but you can stick with lib if you want.